tests/phpunit/tests/feed/rss2.php

```diff
@@ -58,6 +60,18 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 		foreach ( self::$posts as $post ) {
 			wp_set_object_terms( $post, self::$category->slug, 'category' );
 		}
+
+		self::$post_id = $factory->post->create();
+
+		// Create a comment
```

Comment:
```suggestion
		// Create a comment.
```
Period at the end of the comment for consistency with the rest of the codebase.